packages/cli/src/commands/env/command.ts

```diff
@@ -141,12 +141,16 @@ export const pullSubcommand = {
   name: 'pull',
   aliases: [],
   description:
-    'Pull all Development Environment Variables from the cloud and write to a file [.env.local]',
+    'Load all Development Environment Variables from the cloud to a file [.env.local] or a process',
   arguments: [
     {
       name: 'filename',
       required: false,
     },
+    {
+      name: 'process',
```

Comment:
I think command is more accurate than process here
```suggestion
      name: 'command',
```